doc/deployment_examples/sinfonier/feeding_sinfonier.md

```diff
@@ -0,0 +1,408 @@
+Same as subscribing, unsubscription is possible if you don't need more changes or if a new subscription is required to another entity.
+All these procedures use `curl` for transferring some data to `Orion` in `JSON` format.
+
+Let's write an example of use creating a new entity with attributes and a subscription to it.
+Follow the general procedure below for create this entity and receive changes in its values.
+
+* Entity: Book1
+* Attributes: Title, Pages and Price.
+
+1. Check if `Orion` is running properly, asking `Orion` for its version:
+  ```
+  curl -X GET http://localhost:1026/version
+  ```
+
+2. Create a subscription to the entity `Book1`
+  ```
+  (curl localhost:1026/v1/subscribeContext -s -S --header 'Content-type: application/json' --header 'Accept: application/json' -d @- | python -mjson.tool) <<EOF
+  {
+      "entities": [
+          {
+              "type": "Book",
+              "isPattern": "false",
+              "id": "Book1"
+          }
+      ],
+      "attributes": [
+      ],
+      "reference": "http://localhost:5050/notify",
+      "duration": "P1M",
+      "notifyConditions": [
+          {
+              "type": "ONCHANGE",
+              "condValues": [
+                  "title",
+                  "pages",
+                  "price"
+              ]
+          }
+      ],
+      "throttling": "PT5S"
+  }
+  EOF
+  ```
+
+  There are three JSON special parameters when a susbscription is sent:
+  * `attributes`: Refers to the attributes that will be notified to `Cygnus`. If empty, all attributes will be notified to `Cygnus`. If not, only the attributes wrote here will be notified. In this case, if `Orion` receive a change for the attribute `price` in entity `Book1` all the attributes (`title`, `pages` and `price`) will be sent to `Cygnys`.
+  * `condValues`: List of attributes that `Book1` has. When a subscription is done to an entity their attributes must be sent to `Orion`
+  * `reference`: Destination of notifications. In this case we use `http://localhost:5050/notify` but you can configure it with your own destination. This parameter involve that our `Cygnus` must be configured as destination.
+
+  Once sent, you will receive something like that:
+  ```
+  {
+      "subscribeResponse": {
+          "duration": "P1M",
+          "subscriptionId": "your_subscription_id",
+          "throttling": "PT5S"
+      }
+  }  
+  ```
+  The value `subscriptionId` is very important. It's highly recommended to save it in case of you need to unsubscribe to the entity in the future.
+
+  If you receive another answer from `Orion` check your curl: `JSON Parse Error` is a typical error. In that case you will receive this message:
+  ```
+  {
+      "subscribeError": {
+          "errorCode": {
+              "code": "400",
+              "details": "JSON Parse Error",
+              "reasonPhrase": "Bad Request"
+          }
+      }
+  }   
+  ```
+
+3. Append some values in entity `Book1`.
```

Comment:
Create an entity in Orion; we will use the `updateContext` operation of the Orion API with the `APPEND` option for creating.